examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb

```diff
@@ -167,7 +167,7 @@
     "                    Eq, Operator, solve, configuration)\n",
```

Comment:
<p>Flops should be 298 instead of 300?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>